scipy/interpolate/_polyint.py

```diff
@@ -190,43 +190,63 @@ def derivatives(self, x, der=None):
 
     def derivative(self, x, der=1):
         """
-        Evaluate one derivative of the polynomial at the point x
+        Evaluate a single derivative of the polynomial at the point x.
 
         Parameters
         ----------
         x : array_like
             Point or points at which to evaluate the derivatives
 
         der : integer, optional
-            Which derivative to extract. This number includes the
-            function value as 0th derivative.
+            Which derivative to evaluate (default: first derivative).
+            This number includes the function value as 0th derivative.
 
         Returns
         -------
         d : ndarray
-            Derivative interpolated at the x-points. Shape of d is
+            Derivative interpolated at the x-points. Shape of `d` is
             determined by replacing the interpolation axis in the
-            original array with the shape of x.
+            original array with the shape of `x`.
 
         Notes
         -----
-        This is computed by evaluating all derivatives up to the desired
+        This may be computed by evaluating all derivatives up to the desired
         one (using self.derivatives()) and then discarding the rest.
 
         """
         x, x_shape = self._prepare_x(x)
         y = self._evaluate_derivatives(x, der+1)
         return self._finish_y(y[der], x_shape)
 
+    def _evaluate_derivatives(self, x, der=None):
+        """
+        Actually evaluate the derivatives.
+
+        Parameters
+        ----------
+        x : array_like
+            1D array of points at which to evaluate the derivatives
+        der : integer, optional
+            The number of derivatives to evaluate, from 'order 0' (der=1) to order der-1.
```

Comment:
Our linter is a bit lax at the moment, but we still would like to have everything under 80 chars (79 max).